client/src/pages/proposalManagement/ProposalManagement.css

```diff
@@ -0,0 +1,118 @@
+.proposalManagement {
+    display: flex;
+    background: var(--lightGrey);
+}
+
+.proposalManagementLeft{
+    margin-left: 2em;
+    margin-top: 1em;
+    width: 30%;
+}
+
+.proposalManagementRight {
+    width: 70%;
+    padding: 50px;
+    /*background: #ddb122;*/
+    display: flex;
+
+}
+
+.proposalManagementHead{
+    margin-bottom: 2%;
+    /* position: absolute; */
+    /* left: 10px; */
+    font-family: 'IBM Plex Sans', sans-serif;
+    font-weight: bold;
+}
+
+.PMtextboxes{
+    width: 100%;
+    height: 10%;
+}
+
+.titleNewProposal{
+    width : 60%;
+    margin-right: 10%;
+    border-radius: 12px;
+    height:100%;
+}
+
+.dateNewProposal{
+    width: 30%;
+    border-radius: 12px;
+    height: 100%;
+}
+
+.descriptionNewProposal{
+    height: 70%;
+    width: 100%;
+    border-radius: 12px;
+    margin-top: 2%;
+    margin-bottom: 2%;
+}
```

Comment:
For all these textboxes, you should make one class that specifies all the general textbox styling (i.e. the `border-radius: 12px`) and then just give this class to every textbox (i.e. `className="mytextbox specificClass"`) 
Also, I think you need to add a `mytextbox:focus` CSS for when you click in to the textbox to make sure that extra black border doesn't appear